Appraisals

```diff
@@ -1,13 +1,14 @@
-appraise "rails-4" do
-  gem "rails", "~> 4", "< 5"
-  gem "sqlite3", "~> 1.3.6"
-end
-
 appraise "rails-5" do
   gem "rails", "~> 5", "< 6"
   gem "sqlite3", "~> 1.3.6"
 end
 
 appraise "rails-6" do
   gem "rails", "~> 6", "< 7"
+  gem "sqlite3"
+end
+
+appraise "rails-7" do
+  gem "rails", "~> 7", "< 8"
+  gem "sqlite3"
 end
```

Comment:
Do we still need this if we aren't using `appraisal`?

Reply:
I think we can probably kill appraisals entirely. It's a little less ergonomic locally....

```
BUNDLE_GEMFILE=gemfiles/rails_7.gemfile bundle install
BUNDLE_GEMFILE=gemfiles/rails_7.gemfile bundle exec rspec -w
```

But not terrible.  Shall I yank it all the way out?

Reply:
Yeah, let's do it! :)